File: SEM5_CODE/PYTHON/BASIC/13_multiplication_table.py
number = int(input("Enter a number: "))          # Read the table number

for multiplier in range(1, 11):                  # Generate multipliers from 1 to 10
    result = number * multiplier                 # Calculate the table value
    print(number, "x", multiplier, "=", result)  # Display the table line

File: SEM3_CODE/PHP/BASIC/13_multiplication_table.php
<?php
$number = (int) readline('Enter a number: ');                        // Read the table number

for ($multiplier = 1; $multiplier <= 10; $multiplier++) {            // Generate multipliers
    $result = $number * $multiplier;                                 // Calculate the table value
    echo $number . ' x ' . $multiplier . ' = ' . $result . PHP_EOL;  // Display the table line
}
?>
